### PR TITLE
Added banhammer and syndicate banhammer to traitor uplink

### PR DESCRIPTION
**Pull request by Arzion5**
:cl:
add: Added a banhammer to the traitor uplink, which can be found in the "Badass" section of your uplink. You can now give heart attacks to players!
add: Added a syndicate banhammer to the traitor uplink, which is a special variant of the banhammer that deals damage on hit without being obvious.
/:cl:

Another fun gimmicky item. Last PR, I promise. I've been asked to split the PRs, so I split them.

### CODE REVIEW

File: hippiestation/code/game/objects/items/weaponry.dm [hunk 1]
```diff
@@ -28,4 +28,12 @@
 		M.adjustBruteLoss(1)
 		visible_message("<span class='greenannounce'>[user] has scored a critical hit on [M]!</span>")
 		playsound(src, 'sound/arcade/mana.ogg', 50, 1)
-	..()
\ No newline at end of file
+	..()
+
+/obj/item/banhammer/syndicate
+	desc = "A banhammer. Upon closer inspection, it appears to have a red tag around its handle."
+	icon = 'hippiestation/icons/obj/items_and_weapons.dmi'
```
Comment (KayeArray) [inline hippiestation/code/game/objects/items/weaponry.dm:35]: is this the proper filename?

**Review by KayeArray — DISMISSED** [review]

File: hippiestation/code/game/objects/items/weaponry.dm (hunk 1 above)
Comment (Arzion5) [reply hippiestation/code/game/objects/items/weaponry.dm:35]: Yes, that's where all the Hippiestation icons exist

**Comment by ghost** [comment]
Sure +1

**Comment by dreamcrab** [comment]
seems harmless enough +666

**Comment by praisenarsie** [comment]
don't train people to ignore the bwoink, -1

**Comment by KayeArray** [comment]
If people ignore bwoinks just fill them with neurotoxin and see them ignore it then.

**Comment by ghost** [comment]
literally has been an item in the game for ages and you used to be able to get it every round
it's never been an issue

**Comment by praisenarsie** [comment]
man i have to play on hippie more

**Review by KayeArray — APPROVED** [review]

File: hippiestation/code/game/objects/items/weaponry.dm (hunk 1 above)
Comment (ghost) [reply hippiestation/code/game/objects/items/weaponry.dm:35]: Correct

**Comment by dreamcrab** [comment]
sure why not friendly friends +1337

**Comment by KayeArray** [comment]
+1

**Pull request merged**